### DISCUSSION TITLE
OpenSource collection of developer roadmaps (https://devmaps.netlify.app/)

### DISCUSSION THREAD

**Discussion by rajtilakjee** [thuliteio/doks#1015] [Showcase]
[Link](https://devmaps.netlify.app/)

I have been using the Doks template for creating my website DevMaps. It is a free and opensource project, and it a collection of developer roadmaps based on various skills and roles. The roadmaps are prepared using chatGPT.

I am a complete newcomer when it comes to web-dev. However, with the help from @h-enk and a good Samaritan from the Hugo Discord server helped me setup the website. Without their help this would not have been possible.

Please note that I would continue to make changes to the website and the roadmaps, it would be an on-going process. However, this is a good start and so I thought I should share it with everyone here.

**Comment by h-enk** [comment]
Thanks for sharing. Nice site!

**Reply by rajtilakjee** [reply]
Thank you for this great theme and for all the help!
